src/tools/pub/cmd/index.md

```diff
@@ -66,18 +66,24 @@ or any modification of the pubspec.
 
 [`pub get`](/tools/pub/cmd/pub-get)
 : Retrieves the packages that are listed as the dependencies for
-  the application. One of the most commonly used pub commands.
+  the application. If a `pubspec.lock` file already exists, fetches
+  the same version of each dependency (if missing) as listed in the
```

Comment:
Thanks! I tweaked the text just slightly to make it more vague. The "pub-get" page has more info on how the command behaves.